tests/js/language_data.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * language_data.js
+ * ~~~~~~~~~~~~~~~~
+ *
+ * This script contains the language-specific data used by searchtools.js,
+ * namely the list of stopwords, stemmer, scorer and splitter.
```

Comment:
The comments in this file seem a little misleading? This is really just test scaffolding AFAICT

Reply:
Mm, yep - this is an interesting case - it's an evaluated copy of a [template file](https://github.com/sphinx-doc/sphinx/blob/de4ac2fbdebae529a973910e96794645158f0401/sphinx/themes/basic/static/language_data.js_t) from the basic theme.  That's me attempting to make the JavaScript test environment realistic, but it is duplicative (and a bit confusing).